README.adoc

```diff
@@ -130,7 +123,12 @@ include::finish/src/main/java/io/openliberty/guides/microprofile/InventoryConfig
 
 The first `@Inject` annotation injects the Config object, which is request scoped.
 
-The second `@Inject` annotation injects the `io.openliberty.guides.microprofile.inventory.inMaintenance` configuration property, a dynamic injection that uses the `Provider<>` interface, which forces the program to retrieve the `inMaintenance` value just in time.
+[source, java, indent=0]
+----
+include::finish/src/main/java/io/openliberty/guides/microprofile/InventoryConfig.java[tags=build-in-converter]
+----
+
+The second `@Inject` annotation injects the `io_openliberty_guides_inventory_inMaintenance` configuration property, a dynamic injection that uses the `Provider<>` interface, which forces the program to retrieve the `inMaintenance` value just in time.
```

Comment:
We can use a better word than "program"?

Reply:
changed to 'service'